src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,115 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 10;
+    private static final int ONE_INDEX = 1;
```

Comment:
It`s redundant, in this case you don`t need to store this constant

Reply:
> It`s redundant, in this case you don`t need to store this constant

So how I will specify initial capacity?
just 10 and that it?